test/integration/test.js

```diff
@@ -414,30 +414,22 @@ describe('integration', function () {
             );
         });
 
-        it('should NOT capture breadcrumbs from XMLHttpRequests to the Sentry store endpoint', function (done) {
+        it('should NOT denote XMLHttpRequests to the Sentry store endpoint as requiring breadcrumb capture', function (done) {
```

Comment:
Had to change this test because Phantom 2 throws a network error that DOES get collected as a breadcrumb. Better that this test not attempt to make an XHR at all.